### ISSUE TITLE
File name with `(` and `)` return a false and does not scan

### ISSUE THREAD

**Issue by gillien** [kobaltz/clamby#42]
Hello,

When calling  `Clamby.safe?` with a file path which include the character `(` or `)` (my filename is `filename(2021-10-01).pdf`, it returns an error and does not test the file: 

```
> Clamby.safe?(path)
ClamAV 0.103.2/26364/Thu Nov 25 09:20:31 2021
sh: 1: Syntax error: "(" unexpected
 => false
```

I am running it on a Ubuntu 20 server

Thank you!

**Comment by abrom** [comment]
Your issue appears to already be resolved via #37 although it hasn't been released. You'd either need @kobaltz to release an updated gem, or reference the master branch when installing the gem

**Comment by kobaltz** [comment]
v1.6.8 has been released. Sorry for the delay.
